path_oidc.go

```diff
@@ -537,6 +538,17 @@ func (b *jwtAuthBackend) verifyOIDCRequest(stateID string) *oidcRequest {
 	return nil
 }
 
+func isLoopbackAddress(hostname string) bool {
```

Comment:
Nice! One minor nit, I would maybe name this function `isLocalAddr()` or `isLocalhost()` since it also compares against a hostname.